web/src/engine/osk/src/views/floatingOskView.ts

```diff
@@ -66,6 +67,31 @@ export default class FloatingOSKView extends OSKView {
 
     this.headerView = this.titleBar;
 
+    const onListenedEvent = (eventName: keyof EventMap) => {
+      // As the following title bar buttons (for desktop / FloatingOSKView) do nothing unless a site
+      // designer uses these events, we disable / hide them unless an event-handler is attached.
+      let titleBar = this.headerView;
+      if(titleBar && titleBar instanceof TitleBar) {
+        switch(eventName) {
+          case 'showConfig':
+            titleBar.configEnabled = this.listenerCount('showConfig') + this.legacyEvents.listenerCount('configclick') > 0;
+            break;
+          case 'showHelp':
+            titleBar.helpEnabled = this.listenerCount('showHelp') + this.legacyEvents.listenerCount('helpclick') > 0;
+            break;
```

Comment:
As the implementations here suggest, there are two different event types defined; I'd originally been hoping to retire the older KMW events for the buttons and such, but we recently decided not to.  It'd be easy enough to remove the newer-typed events for now, favoring the old ones, until we do a proper redesign.

Either way, I'm not publicly documenting the "newer-typed" ones until we do (or are ready to do) a proper, official API overhaul.